mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/validation/FieldValidator.kt

```diff
@@ -18,8 +18,10 @@ class FieldValidator(
 ) {
     val errors = mutableListOf<FieldError>()
 
-    fun error(error: () -> FieldError) {
-        errors.add(error())
+    fun error(condition: Boolean = true, error: () -> FieldError) {
+        if (condition) {
+            errors.add(error())
+        }
```

Comment:
Denne blir jo helt lik validate men motsatt. Var det for å slippe å negate sjekkene ved refaktorering? Litt redd for at denne gjør det vanskeligere å skjønne om testen er riktig eller snudd, når vi nå har to forskjellige eller?